invenio_admin/ext.py

```diff
@@ -175,15 +179,17 @@ def __getattr__(self, name):
 
 def finalize_app(app):
     """Finalize app."""
+    invenio_admin = app.extensions["invenio-admin"]
+    if entry_point_group := invenio_admin.entry_point_group:
+        invenio_admin.load_entry_point_group(entry_point_group)
```

Comment:
@rerowep thanks for the contribution! It looks good but for the record what was the problem that you were facing and moved the initialization of the models to the `finalize_app`?

Reply:
After updating invenio packages, the invenio command failed randomly with this kind of error:
```
Failed to initialize entry point: EntryPoint(name='invenio_admin', value='invenio_admin:InvenioAdmin', group='invenio_base.apps')
```.
The problem is that invenio admin try to declare some invenio-db(SQLAlchemy) models but sometimes the invenio-db extension is not yet initialized. As you know the order of the flask extensions cannot be forced. I do not understand why it was not the case using previous versions. Maybe you have an idea.

Reply:
Thanks for the clarification!Flask v3 removed some signals that we were relying on to initialize entrypoints and thus we introduced the `finalize_app` hook. I assume is a consequence of that change.